src/test/scala/ch/epfl/bluebrain/nexus/tests/Keycloak.scala

```diff
@@ -0,0 +1,86 @@
+package ch.epfl.bluebrain.nexus.tests
+
+import akka.http.javadsl.model.headers.HttpCredentials
+import akka.http.scaladsl.model.HttpMethods._
+import akka.http.scaladsl.model.HttpRequest
+import akka.http.scaladsl.model.headers.Authorization
+import akka.http.scaladsl.unmarshalling.FromEntityUnmarshaller
+import akka.stream.Materializer
+import ch.epfl.bluebrain.nexus.commons.http.HttpClient.UntypedHttpClient
+import io.circe.Json
+import io.circe.optics.JsonPath._
+import monix.bio.Task
+import scala.concurrent.duration._
+
+trait Keycloak
+
+object Keycloak extends Keycloak {
+
+  import monix.execution.Scheduler.Implicits.global
+
+  private val realmEndpoint =
+    s"http://${System.getProperty("keycloak:8080")}/auth/realms/nexus-tests/protocol/openid-connect/token"
```

Comment:
to make this cleaner, the `System.getProperty("keycloak:8080")` variable could be added to the `TestConfig` case class

Reply:
This is provided by 
https://github.com/Tapad/sbt-docker-compose#to-execute-scalatest-test-cases-against-a-running-instance when running the test, it allows to get the dynamic endpoint for keycloak and delta.

So we can't do much about it.